app/models/agents/pushbullet_agent.rb

```diff
@@ -114,14 +109,17 @@ def create_device
       end
     end
 
-
     def basic_auth
       {basic_auth: {username: interpolated[:api_key].presence || credential('pushbullet_api_key'), password: ''}}
     end
 
     def query_options(event)
       mo = interpolated(event)
-      basic_auth.merge(body: {device_iden: mo[:device_id], type: mo[:type]}.merge(payload(mo)))
+      if mo[:device_id] == "__ALL__"
+        basic_auth.merge(body: {device_iden: "", type: mo[:type]}.merge(payload(mo)))
```

Comment:
i tired both of these, and neither of them worked for some reason, however it does work with the if statement.